gallery/src/components/NxList/NxListPage.tsx

```diff
@@ -0,0 +1,284 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxP, NxCode, NxList, NxTextLink, NxTable } from '@sonatype/react-shared-components';
+import {GalleryDescriptionTile} from '../../gallery-components/GalleryTiles';
+import NxListsExamples from './NxListExamples';
+
+const NxAlertPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>Lists take many forms:</NxP>
+      <NxList bulleted>
+        <NxList.Item>Simple data lists</NxList.Item>
```

Comment:
Rather than repeating the docs from the `nx-list` styles page, you can have a more brief intro here that explains that these are the react components which encapsulate use of those styles.

Reply:
> Rather than repeating the docs from the `nx-list` styles page, you can have a more brief intro here that explains that these are the react components which encapsulate use of those styles.

I've added some brief info about `NxList` and how they're simple React components encapsulating the `.nx-list` class.
ef7e7f07d8f54b1ed605b492c6dc36366534b770